NOpenType/ToPixelRasterizer.cs

```diff
@@ -2,34 +2,37 @@
 
 namespace NRasterizer
 {
-    public class ToPixelRasterizer: IGlyphRasterizer
+    /// <summary>
+    /// Converts from glyph point based values into pixel based ones
+    /// </summary>
+    /// <remarks>
+    /// This is a struct because its super short lived and it prevents the need for an allocation for each character rendered.
+    /// This is internal as it will only ever be used by the <see cref="Renderer"/> and should not be used by external systems.
+    /// </remarks>
+    internal struct ToPixelRasterizer
     {
         private readonly IGlyphRasterizer _inner;
         private readonly float _x;
         private readonly float _y;
-        private readonly int _m;
-        private readonly int _d;
+        private readonly int _scalingFactor;
 
-        public ToPixelRasterizer(float x, float y, int multiplyer, int divider, IGlyphRasterizer inner)
+        public ToPixelRasterizer(int x, int y, int scalingFactor, IGlyphRasterizer inner)
```

Comment:
I think it's clearer if the divider is passed in as well, instead as grabbed as a constant. Le'ts keep it that way.